{
  "Faults": [
    {
      "latitude": 51.521252970352506,
      "type": "LV",
      "reference": "KT0598",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "44c5ca50-2505-4657-9785-80658b74538d",
      "minLongitude": -1.6634,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.6564784346640522,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-02T22:00:00.000+0000",
      "estimatedRestoration": "2023-04-03T01:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT0598'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT0598",
      "maxLongitude": -1.6432,
      "minLatitude": 51.5087,
      "name": "SN4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN4 0DT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6634,
              51.525
            ],
            [
              -1.6432,
              51.525
            ],
            [
              -1.6432,
              51.5087
            ],
            [
              -1.6634,
              51.5087
            ],
            [
              -1.6634,
              51.525
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-02T21:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.525,
      "etrUpdate": true
    },
    {
      "latitude": 50.65473425253871,
      "type": "LV",
      "reference": "KT0889",
      "etaUpdate": true,
      "affectedCustomerCount": 41,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "3fdbbcf1-7c10-4452-acf0-2c1885d5a5cf",
      "minLongitude": -1.1586,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1565405427936075,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-03T01:04:00.000+0000",
      "estimatedRestoration": "2023-04-03T04:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT0889'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT0889",
      "maxLongitude": -1.1484,
      "minLatitude": 50.6535,
      "name": "PO36 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO36 8DA",
        "PO36 8DN",
        "PO36 8FA",
        "PO36 8HW",
        "PO36 8HX",
        "PO36 8HY",
        "PO36 8HZ",
        "PO36 8JB",
        "PO36 8JD",
        "PO36 8JG",
        "PO36 8JJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1586,
              50.6614
            ],
            [
              -1.1484,
              50.6614
            ],
            [
              -1.1484,
              50.6535
            ],
            [
              -1.1586,
              50.6535
            ],
            [
              -1.1586,
              50.6614
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-03T02:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.6614,
      "etrUpdate": true
    },
    {
      "latitude": 51.53374448027131,
      "type": "LV",
      "reference": "KT0903",
      "etaUpdate": true,
      "affectedCustomerCount": 11,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d91e9b30-59ef-42b4-bea6-252229f61c9e",
      "minLongitude": -1.0794,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0770799097985422,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-03T01:42:00.000+0000",
      "estimatedRestoration": "2023-04-03T04:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT0903'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT0903",
      "maxLongitude": -1.0751,
      "minLatitude": 51.5319,
      "name": "RG8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG8 0PR",
        "RG8 0QJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0794,
              51.5354
            ],
            [
              -1.0751,
              51.5354
            ],
            [
              -1.0751,
              51.5319
            ],
            [
              -1.0794,
              51.5319
            ],
            [
              -1.0794,
              51.5354
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5354,
      "etrUpdate": true
    },
    {
      "latitude": 51.53318982423306,
      "type": "LV",
      "reference": "KT0715",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0ed286c1-9a7c-4e76-a629-64353d2a5ea1",
      "minLongitude": -1.0872,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0797294437320042,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-03T01:25:00.000+0000",
      "estimatedRestoration": "2023-04-03T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KT0715'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KT0715",
      "maxLongitude": -1.078,
      "minLatitude": 51.5318,
      "name": "RG8 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG8 0PP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0872,
              51.5368
            ],
            [
              -1.078,
              51.5368
            ],
            [
              -1.078,
              51.5318
            ],
            [
              -1.0872,
              51.5318
            ],
            [
              -1.0872,
              51.5368
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-02T21:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5368,
      "etrUpdate": true
    }
  ]
}
